python/ray/tune/tune.py

```diff
@@ -381,7 +385,8 @@ def run(
 
     if progress_reporter is None:
         if IS_NOTEBOOK:
-            progress_reporter = JupyterNotebookReporter(overwrite=verbose < 2)
+            progress_reporter = JupyterNotebookReporter(
+                overwrite=not has_verbosity(Verbosity.TRIAL_NORM))
```

Comment:
hmm I couldn't quite read what this means without looking at the underlying implementation.